writeimage.sh

```diff
@@ -152,9 +184,22 @@ if [ -n "$IP" ] && [ -n "$GW" ] && [ -n "$DNS" ]; then
     echo "static_dns=\"$DNS\"" >> $conf
 fi
 
-msg "unmounting sdcard"
-sync
-umount $BOOT
-rmdir $BOOT
+if [ `uname` == "Darwin" ]; then
+	# copy created files over to the disk
+	DISK_PARTITION=`diskutil info $BOOT_DEV | grep "Mount Point:" | sed 's/^[^/]*//' | sed 's/ /\\ /g'`
+
+	echo "moving $BOOT/* to $DISK_PARTITION"
+
+	mv -v "$BOOT"/* "$DISK_PARTITION"
```

Comment:
I am not sure.   When I tried to mount the partition to the location of $BOOT on my mac, I would just get errors. I tried:

`mount $BOOT_DEV $BOOT`
`mount -ft msdos $BOOT_DEV $BOOT`

I also tried replacing "msdos" in the second option above with other types of file system designations, but nothing would work to mount it.  However, I do see now that in an old version of writeimage.sh you used the mount_msdos command.  I didn't try that option.

The way I did the mounting (using hdiutil) requires the name of the volume according to the MAC file system (like "/Volumes/NO NAME"), so this is why I used the $DISK_PARTITION variable.  So in this case, the files are written to a directory that is not on the sd card ($BOOT), then copied to the sd card ($DISK_PARTITION).